spring-cloud-gcp-starters/spring-cloud-gcp-starter-sql/src/main/java/org/springframework/cloud/gcp/sql/GcpCloudSqlProperties.java

```diff
@@ -68,6 +70,14 @@ public void setRegion(String region) {
 		this.region = region;
 	}
 
+	public String getInstanceConnectionName() {
+		return this.instanceConnectionName;
+	}
+
+	public void setInstanceConnectionName(String instanceConnectionName) {
+		this.instanceConnectionName = instanceConnectionName;
+	}
+
```

Comment:
That's a good point. In fact, every starter should probably specify their own project ID in case that service is running on a different project that's not the default one.